src/Mariuzzo/LaravelJsLocalization/Commands/LangJsCommand.php

```diff
@@ -95,6 +98,7 @@ protected function getOptions()
     {
         return [
             ['compress', 'c', InputOption::VALUE_NONE, 'Compress the JavaScript file.', null],
+            ['source', 's', InputOption::VALUE_REQUIRED, 'Specifying a custom source folder', null],
```

Comment:
Right now `-s` key requires value, I think that's correct behavior. Use default path with empty `-s` key value not required on my opinion. If user want to use default path - there is no need to define `-s` key.